lib/json5.js

```diff
@@ -350,6 +396,17 @@ JSON5.parse = (function () {
             }
         },
 
+        space = function () {
+// like `white`, but this only skip whitespace
+            while(ch) {
+                if (~ws.indexOf(ch)) {
```

Comment:
I know that `~-1 == 0 == false` but I don't see this used very often, so let's simplify this to `ws.indexOf(ch) >= 0` for readability.